packages/graphql/tests/tck/issues/missing-custom-cypher-on-unions.ts

```diff
@@ -0,0 +1,236 @@
+/*
```

Comment:
The name of the test file does not contains the suffix `.test.`

Reply:
Good catch